package.json

```diff
@@ -48,10 +48,10 @@
     "lucide-react": "^0.236.0",
     "mini-svg-data-uri": "^1.4.4",
     "mixpanel-browser": "^2.45.0",
-    "next": "^13.2.4",
-    "react": "^18.2.0",
+    "next": "^14.2.3",
```

Comment:
Is there a minimum node version we need with next14 ?

Reply:
Node 18, which is the one that we have on package.json